context/build-utils/webpack.common.js

```diff
@@ -61,7 +61,7 @@ const config = {
       },
       {
         test: /\.svg$/,
-        use: { loader: '@svgr/webpack' },
+        use: ['@svgr/webpack', 'url-loader'],
```

Comment:
I'm guessing this is related to the changes above:
```diff
-import Logo from 'images/hubmap-logo.svg';
+import { ReactComponent as Logo } from 'images/hubmap-logo.svg';
```
It is more verbose... What is better about this approach?

Reply:
I wanted to be able to load svgs by url and as react components. We could either do it this way which requires the highlighted syntax when importing as a component or I could use a distinct file pattern for svg files I intend to load as components such as `file.react.svg` or similar and have separate rules in the webpack config for each svg file pattern.

Reply:
Per our Zoom conversation, the new svg bundling approach is fine.